pkg/gce-cloud-provider/compute/fake-gce.go

```diff
@@ -422,6 +421,42 @@ func (cloud *FakeCloudProvider) getGlobalSnapshotURI(snapshotName string) string
 		snapshotName)
 }
 
+type FakeBlockingCloudProvider struct {
+	*FakeCloudProvider
+	snapshotToExec chan SnapshotSourceAndTarget
+	readyToExecute chan struct{}
```

Comment:
nit: naming. `readyToExecute` doesn't really tell me much about who is ready to execute what. Maybe `createSnapshotCalled` is more accurate?